tests/utils.py

```diff
@@ -87,3 +87,21 @@ def check_server_ready(protocol="grpc"):
         output = buf.getvalue()
     output = json.loads(output)
     return output["ready"]
+
+
+# Context Manager to start and kill a mock server running in background and used by testing functions
+class ScopedTritonServer:
+    def __init__(self, repo=None, mode="local", timeout=60):
+        self.repo = repo
+        self.mode = mode
+        self.timeout = timeout
+
+    def __enter__(self):
+        run(["remove", "-m", "all"])
```

Comment:
I think clearing the model repository on __enter__ is the cause for some of the job failures. It's doing something like:
```
self.import() # create model repo
with utils.ScopedTritonServer:
  # clears model repo
  # start server
  # error: model repo not found - it was removed
```